core/block/editor/bookmark/bookmark.go

```diff
@@ -201,15 +202,21 @@ func (b *sbookmark) MigrateBlock(bm bookmark.Block) error {
 		return nil
 	}
 
-	pageId, _, err := b.bookmarkSvc.CreateBookmarkObject(bm.ToDetails(), func() *model.BlockContentBookmark {
+	bookmarkContent := func() *model.BlockContentBookmark {
 		return content
-	})
+	}
+
+	if content.Description == "" && content.FaviconHash == "" && content.Title == "" {
+		bookmarkContent = b.bookmarkSvc.FetchBookmarkContent(content.Url)
```

Comment:
Maybe do it in bookmark syncer? This code is called only in import

Reply:
I think we have to delete MigrateBlock completely because it's called only in state migration. Seems like import is the only case when something meaningful is happening here

Reply:
Removed migration completely